lua/ulx/modules/sh/freeze.lua

```diff
@@ -0,0 +1,28 @@
+CATEGORY_NAME = "CFC"
+
+local function freezeProps( callingPlayer, targetPlayers ) 
+    local entities = ents.FindByClass( "prop_*" )
```

Comment:
Would we want to freeze other things, like e2s, wire chips, etc.?

Reply:
Done